exercises/series/src/test/java/SeriesTest.java

```diff
@@ -17,145 +17,117 @@
     public ExpectedException expectedException = ExpectedException.none();
 
     @Test
-    public void hasDigitsShort() {
-        Series series = new Series("01234");
```

Comment:
The indentation appears to have gone a bit wrong in this file. Please make sure everything is indented correctly, using four spaces for each indentation 🙂